tools/dataset_generator.py

```diff
@@ -163,7 +163,7 @@ def save_demo(demo, example_path):
 
     # Save the low-dimension data
     with open(os.path.join(example_path, LOW_DIM_PICKLE), 'wb') as f:
-        pickle.dump(demo, f)
+        pickle.dump([obs.get_low_dim_data() for obs in demo], f)
```

Comment:
Thanks for the PR!
This line should be reverted though, as if you call obs.get_low_dim_data, it actually squashes all of it into one big bumpy array, which means we loose the semantics of what those values mean.
The existing code works fine, and it doesn't a tually pickle any images, as if you look above, you'll see that we set all of the image data to None.

Reply:
Oh ok, my bad :smile:. I thought it was pickling the whole demo. Didn't catch that None setting. Will revert it then :+1:

Reply:
Alright, reverted it on bd2027a commit.